docs cross-site route #1312 (#1488)
infinispan/infinispan-operator: documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc

.Prerequisites

* Determine a suitable expose type for cross-site replication. +

> If you use an {openshiftshort} `Route` you must add a keystore and encrypt RELAY messages between clusters.

Reviewer: ```suggestion
If you use an {openshiftshort} `Route` you must add a keystore with the certificates for the TLS traffic.
```
